workbook/components/session_service.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+from django.utils import timezone
+
+from workbook.models.models import Session
+
+
+class SessionService:
+    def get_time_difference(self, session_start_time):
+        time_difference = timezone.now() - session_start_time
+        return time_difference.total_seconds() / 60
+
+    def is_valid(self, session_start_time):
+        return self.get_time_difference(session_start_time) < 1
+
+    def get(self, user_id):
+
+        session = Session.objects.filter(user_id=user_id).order_by('-token_start_time').first()  # '-' desc
+
+        if session and self.is_valid(session.token_start_time):
+            return session
+
+        return None
+
+    def update(self, session):
+        session.token_start_time = timezone.now()
+        session.save()
+
+    def create(self, user_id):
+        return Session.objects.create(token=uuid.uuid1(), user_id=user_id)
```

Comment:
you can add 
```suggestion
        return Session.objects.create(token=uuid.uuid1().hex, user_id=user_id)
```
so the token doesn't have hyphens
559ce38862b211efac0c3a3582ecc5b9 instead of 7f3b1f61-d097-4f55-9ac2-69bdc6082b1c